web-ui/src/context/selectors.js

```diff
@@ -49,16 +49,33 @@ export const selectOrderedSkills = createSelector(selectSkills, (skills) =>
   })
 );
 
-export const selectProfileMap = createSelector(
+export const selectCurrentMembers = createSelector(
   selectMemberProfiles,
-  (memberProfiles) => {
-    if (memberProfiles && memberProfiles.length) {
-      memberProfiles = memberProfiles.reduce((mappedById, profile) => {
-        mappedById[profile.id] = profile;
+  (memberProfiles) =>
+    memberProfiles
+      ?.filter((profile) => {
+        const today = new Date();
+        today.setHours(0, 0, 0, 0);
+        return (
+          profile.terminationDate === null ||
+          profile.terminationDate === undefined ||
+          today <= new Date(profile.terminationDate)
+        );
+      })
+      .sort((a, b) => a.lastName.localeCompare(b.lastName))
+);
+
+export const selectProfileMap = createSelector(
+  selectCurrentMembers,
+  (currentMembers) => {
+    if (currentMembers && currentMembers.length) {
+      currentMembers = currentMembers.reduce((mappedById, member) => {
+        mappedById[member.id] = member;
+        console.log(mappedById);
```

Comment:
don't forget this lil homie

Reply:
Can't believe I forgot another one of these. Thanks!